### PR TITLE
Minor preprocessor bug fixes

### PR DESCRIPTION
**Pull request by s-alexey**
This bugs clauses exceptions in Preprocessor and Bag of Words widgets.

### CODE REVIEW

**Comment by s-alexey** [comment]
This fails by problem in `Corpus.copy()` method (it call `init` without arguments). It doesn't relate to this PR.

**Comment by nikicc** [comment]
Sure. We are fixing the `copy` in a PR #36. Although, this PR should not break anything, I suggest waiting until #36 is merged so we re-can run the tests on this one.

**Comment by kernc** [comment]
@s-alexey Could you include a unit test that covers your change?

File: orangecontrib/text/preprocess.py [hunk 1]
```diff
@@ -57,16 +57,21 @@ def __call__(self, data):
...
     """
     def __init__(self, trans, lowercase=True, name='Stemmatizer'):
         """
             :param trans: The method that will perform transformation on the tokens.
+            :type trans: Callable (function or method)
             :param name: The name of the transformation object.
+            :type name: str
             :return: :class: `orangecontrib.text.preprocess.Stemmatizer`
         """
+        if not hasattr(trans, '__call__'):
```
Comment (kernc) [inline orangecontrib/text/preprocess.py]: Python >=3.2 has `callable()` again.

File: orangecontrib/text/widgets/owpreprocess.py [hunk 2]
```diff
@@ -65,6 +63,9 @@ def apply(self):
         # TODO change this to custom stopwords
         if self.remove_stpwrds:
             sw = 'english'
+        else:
+            sw = None
```
Comment (kernc) [inline orangecontrib/text/widgets/owpreprocess.py]: I prefer simple conditional value expressions like this one with a ternary operator:
```py
sw = 'english' if self.remove_stpwrds else None  # sic!
```
Four lines vs. one for something trivial.

Comment (nikicc) [inline orangecontrib/text/widgets/owpreprocess.py]: Me too. But this is going to be replaced by David's PR #21 anyway.

File: orangecontrib/text/preprocess.py (hunk 1 above)
Comment (nikicc) [inline orangecontrib/text/preprocess.py]: Did not know this. Will fix someday :smile:

**Pull request merged**